vsintegration/src/FSharp.Editor/Commands/XmlDocCommandService.fs

```diff
@@ -58,76 +60,80 @@ type internal XmlDocCommandFilter(wpfTextView: IWpfTextView, filePath: string, w
 
                     match XmlDocComment.IsBlank lineWithLastCharInserted with
                     | Some i when i = indexOfCaret ->
-                        asyncMaybe {
```

Comment:
This one might be a problem, due to how Async.StartImmediate works, will revert to the asyncMaybe.